projects/observability/src/pages/explorer/explorer-dashboard-builder.ts

```diff
@@ -606,9 +604,25 @@ export interface ExplorerGeneratedDashboard {
   onReady(dashboard: Dashboard): void;
 }
 
-interface AttributeValue {
-  type: string;
-  attribute: string;
+interface ModelSpecificationJson extends ModelJson {
```

Comment:
looks good here. Is value optional though? do we have columns with no value?

Reply:
needs to be mandatory

Reply:
updated